react-forms/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState, ChangeEvent, FormEvent } from 'react';
+
+export function RegistrationFormControlled() {
+  const [username, setUsername] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+
+  const handleUsernameChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    console.log(' Controlled Form data:', { username, password });
+  };
+
+  return (
+    <>
+      {' '}
```

Comment:
This was probably put here by prettier when it formatted your file. You can delete it.